.github/workflows/test-site.yml

```diff
@@ -0,0 +1,33 @@
+name: Test site
```

Comment:
To stay consistent with our other workflow names, we should capitalize each word in the workflow name

`Test Site`